src/main/java/games/strategy/triplea/ui/UnitChooser.java

```diff
@@ -38,37 +37,33 @@
   private JTextArea title;
   private int total = -1;
   private final JLabel leftToSelect = new JLabel();
-  private final GameData data;
```

Comment:
This field was no longer used after removing the `GameData` parameter from the various constructors.